contrib/node/src/python/pants/contrib/node/targets/node_module.py

```diff
@@ -24,10 +30,15 @@ def __init__(self, sources=None, address=None, payload=None, **kwargs):
     # tasks.  The reality is likely to be though that both pants will never cover all cases, and a
     # back door to execute new tools during development will be desirable and supporting conversion
     # of pre-existing package.json files as node_module targets will require this.
+    package_manager = 'yarnpkg' if package_manager == 'yarn' else package_manager
```

Comment:
The problem here is that the full name of this manager is yarnpkg. However, for most node developers, they only know yarn as the abbreviation of this package. yarn is ambiguous with the hadoop map-reduce. So I use yarnpkg in all the code but I still want to allow developers to use the abbreviation "yarn" when they are configuring their project.